test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -193,34 +263,285 @@ def __init__(self):
             def forward(self, x):
                 return self.sigmoid(x)
 
-        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(SigmoidModel(), (x,))
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(self, SigmoidModel(), (x,))
+
+    @pytorch_test_common.skip_dynamic_fx_test(
+        "_aten_convolution_onnx: _add_attribute_to_torchscript_node()"
+        " parameter value=[None, None] violates type hint"
+        "typing.Union[float, int, str, bytes, typing.Sequence[float],"
+        " typing.Sequence[int], torch.Tensor], as [None, None]:"
+    )
+    def test_shufflenet_v2_dynamic_axes(self):
+        model = torchvision.models.shufflenet_v2_x0_5(pretrained=False)
```

Comment:
How does this pass CI ... Seems torchvision is not imported from anywhere?

Reply:
Isn't this skipped?

Reply:
🫢

Reply:
no worries, I'm adding a skip decorator for torchvision if uninstalled in the other PR